tests/ecosystem/upgrade/test_logging_upgrade.py

```diff
@@ -0,0 +1,165 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.cluster import CephCluster, CephHealthMonitor
+from ocs_ci.utility.deployment_openshift_logging import (
+    check_health_of_clusterlogging, get_clusterlogging_subscription
+)
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources.pod import get_all_pods
+from ocs_ci.framework.pytest_customization.marks import (
+    post_ocp_upgrade
+)
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.utility.utils import (
+    TimeoutSampler, get_ocp_version, run_cmd
+)
+
+logger = logging.getLogger(__name__)
+
+
+def check_cluster_before_after_upgrade():
```

Comment:
Suggest changing the name to "check_cluster_logging" as this can be called before and after the upgrade.

Reply:
Done